lib/server.coffee

```diff
@@ -43,26 +44,28 @@ class Server
   ###
   _setupRoutes: ->
     @_names()
-    if isTruthy process.env['LOCAL_TESTING']
-      @_setupLocalTestingRoutes()
-    else
+    if isTruthy process.env['CLOUDMINE']
```

Comment:
This basically exposes to the client the environment variable. Is there a mechanism to make this non-mutable? I feel like this should come from a config environment. Kinda scares me that we make this visible to the implementer.